multiaccounts/accounts/database.go

```diff
@@ -559,6 +560,9 @@ func (db *Database) GetSettings() (Settings, error) {
 		&s.AutoMessageEnabled,
 	)
 
+	communitiesSettings, err := db.GetCommunitiesSettings()
+	s.CommunitiesSettings = communitiesSettings
```

Comment:
If there's an `err`, it end up being overwritten by a succesful `db.getCommunitiesSetting()` response

```suggestion
        if err != nil {
            return s, err
        }

	communitiesSettings, err := db.GetCommunitiesSettings()
	s.CommunitiesSettings = communitiesSettings
```

Reply:
Good catch!!